Tests/ComposableArchitectureTests/EffectCancellationTests.swift

```diff
@@ -5,10 +5,10 @@ import XCTest
 
 final class EffectCancellationTests: XCTestCase {
   var cancellables: Set<AnyCancellable> = []
-
-  override func setUp() {
-    super.setUp()
-    resetCancellables()
+  
+  override func tearDown() {
+    super.tearDown()
+    cancellables.removeAll()
```

Comment:
We _do_ like explicit `self` though. 😆
```suggestion
    self.cancellables.removeAll()
```